pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -220,6 +221,16 @@ func (c *Cluster) Start() error {
 }
 
 func (c *Cluster) startProvisioningOverPVCs(config *provisionConfig) {
+	//check k8s version
+	k8sVersion, err := k8sutil.GetK8SVersion(c.context.Clientset)
+	if err != nil {
+		logger.Warningf("error finding Kubernetes version. %+v", err)
+		return
+	}
+	if !k8sVersion.AtLeast(version.MustParseSemantic("v1.13.0")) {
+		logger.Warningf("skipping OSD on PVC provisioning. Minimum Kubernetes version required: 1.13.0. Actual version: %s", k8sVersion.String())
```

Comment:
We need to think about the right time to log this message.
- If the user doesn't have any settings in the CR for provisioning OSDs on PVs, we shouldn't even need to add this check. Should we add this check around line 252 below after we check the volume sources?
- While we're fixing the warnings, I think the one line 247 should just be an info and could be simplified to: `No volume sources specified for creating OSDs on PVCs`. The rest of the message doesn't even make sense to me now.

Reply:
Ditto Travis' comment about this being an error message also.

Reply:
> We need to think about the right time to log this message.
> 
> * If the user doesn't have any settings in the CR for provisioning OSDs on PVs, we shouldn't even need to add this check. Should we add this check around line 252 below after we check the volume sources?
yes, it would be more appropriate to add this check after checking the volume sources. Updated the code with this change.
> * While we're fixing the warnings, I think the one line 247 should just be an info and could be simplified to: `No volume sources specified for creating OSDs on PVCs`. The rest of the message doesn't even make sense to me now.
Updated the message. Should this still be a `warning` or just an `info`